geomesa-tools/bin/install-hadoop-accumulo.sh

```diff
@@ -49,6 +52,7 @@ else
             "${base_url}commons-logging/commons-logging/${com_log_version}/commons-logging-${com_log_version}.jar"
             "${base_url}com/google/guava/guava/${guava_version}/guava-${guava_version}.jar"
             "${base_url}org/apache/commons/commons-vfs2/${commons_vfs2_version}/commons-vfs2-${commons_vfs2_version}.jar"
+            "${base_url}org/apache/htrace/htrace-core/${htrace_version}/htrace-core-${htrace_version}.jar"
```

Comment:
I actually don't know, as I'm using Accumulo 1.7.0. I suppose that the "extra" dependency would be ignored in case Accumulo 1.6 is used, but I agree that it would be better to test it before accepting the pull-request!